$NetBSD: patch-src_runtime_Config.generic-bsd,v 1.1 2019/11/10 16:58:31 he Exp $

Collect common BSD settings here.
Add to, not set variables.

--- src/runtime/Config.generic-bsd.orig	2019-11-08 20:59:57.163942965 +0100
+++ src/runtime/Config.generic-bsd	2019-11-08 21:01:16.038801843 +0100
@@ -0,0 +1,16 @@
+# -*- makefile -*- for the C-level run-time support for SBCL
+
+# This software is part of the SBCL system. See the README file for
+# more information.
+#
+# This software is derived from the CMU CL system, which was
+# written at Carnegie Mellon University and released into the
+# public domain. The software is in the public domain and is
+# provided with absolutely no warranty. See the COPYING and CREDITS
+# files for more information.
+
+OS_SRC += bsd-os.c
+
+ifdef LISP_FEATURE_SB_CORE_COMPRESSION
+  OS_LIBS += -lz
+endif
